package.json

```diff
@@ -26,21 +29,39 @@
     "angular": "^1.7.9",
     "angular-animate": "1.7.9",
     "angular-cookies": "1.7.9",
+    "angular-elastic": "^2.5.1",
     "angular-route": "1.7.9",
     "angular-sanitize": "1.7.9",
+    "esprima": "^3.1.3",
     "intersection-observer": "^0.7.0",
     "jquery": "^3.4.1",
+    "js-yaml": "^3.13.1",
+    "monaco-editor": "^0.18.1",
+    "ng-showdown": "^1.1.0",
+    "showdown": "^1.9.1",
     "underscore": "^1.9.2"
   },
   "devDependencies": {
     "angular-mocks": "1.7.2",
+    "css-loader": "^3.4.2",
+    "file-loader": "^5.0.2",
     "jsdom": "^15.2.0",
     "karma": "^4.4.1",
     "karma-cli": "^2.0.0",
     "karma-coverage": "^2.0.1",
     "karma-jasmine": "^3.1.0",
     "karma-jsdom-launcher": "^8.0.0",
     "karma-spec-reporter": "^0.0.32",
-    "onchange": "^6.1.0"
+    "mini-css-extract-plugin": "^0.9.0",
+    "monaco-editor-webpack-plugin": "^1.7.0",
+    "node-sass": "^4.13.1",
+    "onchange": "^6.1.0",
+    "sass-loader": "^8.0.2",
+    "shell-loader": "git@github.com:tadam313/shell-loader.git#add-script-options",
```

Comment:
I had to make a change in there to being able to pass options to node `exec`. I pushed my changes back: https://github.com/localjo/shell-loader/pull/1